build_tools/vale/vale_styles/Consensys/Inclusive.yml

```diff
@@ -0,0 +1,15 @@
+extends: substitution
+message: Consider using %s instead of "%s"
+link: https://github.com/Consensys/doc.common/wiki/Documentation-Style-Guide#5-be-inclusive
+ignorecase: true
+level: warning
+swap:
+  'black[ \-]?list(?:ing)?s?': '"denylist", "excludelist", or "blocklist"'
+  'white[ \-]?list(?:ing)?s?': '"allowlist", "trustlist", or "safelist"'
+  'gray[ \-]?list(?:ing)?s?': '"provisional" or "temporary" list'
+  'masters?': '"main", "parent" or "primary"'
+  'slaves?': '"replica" or "subordinate"'
+  'm[ae]n': '"person"'
+  'natives?':  'a more precise term like "built-in"'
+  'first[ \-]?class citizens?': '"core feature" or "top-level"'
```

Comment:
what about second-class citizens

Reply:
How is it used in computers context?

Reply:
I can't think of an example in besu but eg in meetings, people not in the room "nobody is a second-class citizen"

Reply:
Ok, so maybe we don't need it right now. I suggest we start with this small list and be very vigilant during reviews and we add new terms when required.